### ISSUE TITLE
Repeated creation and deletion of LuaScripts results in lua stack overflow

### ISSUE THREAD

**Issue by webermm** [COVESA/ramses-logic#37]
The following test fails in debug builds but not in release builds with the error message:

[sol3] An error occurred and panic has been invoked: stack overflow (not enough space left on Lua stack to push valuees)

```

TEST(ramses_logic, repeat_create_destroy_simple_script) {
	rlogic::LogicEngine logicEngine;

	std::string scriptText = R"(
function interface()
    OUT.out_float = FLOAT
end

function run()
end
)";
	// crashes in debug build with 10000 iterations although ok with 100 iterations
	for (unsigned i = 0; i < 10000; i++) {
		auto* script = logicEngine.createLuaScript(scriptText);
		ASSERT_TRUE(script != nullptr);
		ASSERT_TRUE(logicEngine.destroy(*script));
	}
}

```

Tested with LogicEngine Version 0.13.0

**Comment by violinyanev** [comment]
Thanks @webermm for reporting! Will look into it :)

**Comment by violinyanev** [comment]
@webermm fixed in v0.15.0. Can you confirm the fix works for you?

**Comment by AchimStremplat** [comment]
Yes - this works. I am now able to open scenes with many scripts in debug builds without an error message.
